function.k

```diff
@@ -373,18 +459,27 @@ rule <k> pcsContractPart(C:Id, function F:Id (P1:FParameters) .FunQuantifiers
                <ctname> Cn:Id </ctname>
 	       ...
            </contractInstance> 
+
+  */   
+    
+      rule <k> functionCall(Sender:Int; C:Int ; F:Id ; Es:Values ; M:Msg) => createTransaction(.List) ~> functionCall(F;Es) ~> returnContext(C) ... </k>
+        <contractStack>  (.List => ListItem(C))  ListItem(-1)   </contractStack>
+         <Msg> M1=>M </Msg>
+         <MsgStack> (.List => ListItem(M1)) ... </MsgStack>
+         <functionStack> (.List => ListItem(F)) ... </functionStack>
        
-       rule <k> functionCall(Sender:Int; C:Int ; F:Id ; Es:Values ; M:Msg); => FunCall(C:Int, F:Id, Es:Values,M:Msg) ~> . ... </k>
+       
+       rule <k> functionCall(Sender:Int; C:Int ; F:Id ; Es:Values ; M:Msg) => /*FunCall(C:Int, F:Id, Es:Values,M:Msg)*/ createTransaction(L:List) ~> functionCall(F;Es) ~> returnContext(C)  ... </k>
         <contractStack>  (.List => ListItem(C)) L:List ListItem(-1)   </contractStack>
          <Msg> M1=>M </Msg>
          <MsgStack> (.List => ListItem(M1)) ... </MsgStack>
          <functionStack> (.List => ListItem(F)) ... </functionStack>
-       
+       /*
          <contractInstance>
      	       <ctId> C </ctId>
                <ctname> Cn:Id </ctname>
 	       ...
-           </contractInstance> 
+           </contractInstance> */
```

Comment:
Again, this looks like it's not needed anymore. Remove if you think you don't need it.